### ISSUE TITLE
Train language model in both directions

### ISSUE THREAD

**Issue by hlibbabii** [hlibbabii/log-recommender#20]
Currently when training the model, we are feeding code to it from left to right.

It would be good to try doing it additionally from right to left

**Comment by hlibbabii** [comment]
seems to be supported by torch out of the box

**Comment by hlibbabii** [comment]
compare cross entropy loss for 1 dir and 2
